docs/create/huggingface.md

```diff
@@ -31,7 +31,17 @@ Use the `mk_truss` command to package your model into a Truss.
 ```python
 from truss import mk_truss
 
-mk_truss(model, target_directory="huggingface_truss")
+tr = mk_truss(model, target_directory="huggingface_truss")
 ```
 
 Check the target directory to see your new Truss!
+
+### Serve the model
+
+To get a prediction from the Truss, try running:
+
+```python
+tr.docker_predict({"inputs": "TODO"})
```

Comment:
Is this input a TODO or are you trying to complete a sentence starting with "TODO"